packages/utils/src/peripherals/gelato.ts

```diff
@@ -4,7 +4,8 @@ import { BigNumber } from "ethers";
 import { Logger } from "../logging/logger";
 import { jsonifyError } from "../types";
 
-export const GELATO_SERVER = "https://relay.gelato.digital";
+// export const GELATO_SERVER = "https://relay.gelato.digital";
+export const GELATO_SERVER = "https://relay.gelato.digital2";
```

Comment:
We should just allow an override through the config similar to the other relayer.